.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "mauth-protocol-test-suite"]
+	path = mauth-protocol-test-suite
+	url = git@github.com:mdsol/mauth-protocol-test-suite.git
```

Not sure for AppVeyor, but we need to change this URL for Travis

```suggestion
	url = https://github.com/mdsol/mauth-protocol-test-suite.git
```

https://github.com/mdsol/mauth-client-ruby/pull/45#pullrequestreview-553145146

---

Thanks. Yes Without "https" it said permission denied while cloning "mauth-protocol-test-suite".
Therefore, Updated to "https"